backend/module/chaos/serverexperimentation/rtds/rtds_cache_test.go

```diff
@@ -21,7 +21,12 @@ import (
 	serverexperimentation "github.com/lyft/clutch/backend/api/chaos/serverexperimentation/v1"
 )
 
-func createAbortExperiment(t *testing.T, upstreamCluster string, downstreamCluster string, faultPercent float32, httpStatus int32) *experimentation.Experiment {
+func createAbortExperiment(t *testing.T, upstreamCluster string, downstreamCluster string, faultPercent float32, httpStatus int32, isExternal bool) *experimentation.Experiment {
```

Comment:
same comment as somewhere else - lets pass a faultinjectiontype as opposed to bool here to make it possible for us to test cases for when faultinjectiontype is equal to unspecified